package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "freedom-accounts-util",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
should be 1.1.0.. since we've added a new option

Reply:
Done. PR desciption updated as well.